src/vs/platform/environment/node/argv.ts

```diff
@@ -217,6 +219,38 @@ export function buildVersionMessage(version: string | undefined, commit: string
 	return `${version || localize('unknownVersion', "Unknown version")}\n${commit || localize('unknownCommit', "Unknown commit")}\n${process.arch}`;
 }
 
+export function buildTelemetryMessage(appRoot: string, extensionsPath: string): string {
+	try {
```

Comment:
This needs to be an `async` function. Every single `*Sync` call here should be an `await *` call.

Reply:
Is there a reason this should be async? I was just kind of modeling off of how we build the help message and that isn't async so I figured this didn't need to be either as the user is going to want for the command to finish anyways.

Reply:
Oh yeah, totally right, sorry about that!